dockerfiles/README.md

```diff
@@ -0,0 +1,216 @@
+# Dockerfile Next Steps 
+Table of Contents
+
+[Understanding ENTRYPOINT and CMD](#entrypoint-and-cmd)
+
+## Understanding ENTRYPOINT and CMD
+
+### What's an ENTRYPOINT? What's a CMD? 
+
+#### ENTRYPOINT 
+
+An `ENTRYPOINT` allows you to configure a container that will run as an executable. 
```

Comment:
In previous lectures we already cover running exec/shell forms, and how CMD works. How about we start with stuff like:
- you remember CMD? that's the thing the container runs on start
- There's also ENTRYPOINT, which can also run things on start, but how do they work together?
- Everytime you start a container, docker takes ENTRYPOINT and CMD and just combines them into one long command with a space between them.
- show excalidraw of ENTRYPOINT + [space] + CMD
- Then we dive into why we would want this (the two main use cases of cli tools and scripts)

Reply:
Sounds concise and I like the excalidraw idea.